wiki/Beatmap/Pattern/common/en.md

```diff
@@ -0,0 +1,9 @@
+---
+stub: true
+---
+
+## Common Pattern types
+
+Index for [Patterns](..) that you can occur in every [Game mode](/wiki/Game_mode)
```

Comment:
While I agree that I can drop the common patters for now since there is only one for now, the description of the index pages were untouched and I only added the newer patterns to them, but I can update the description as well I suppose

Reply:
dropped common patterns